documentation/Functions.md

```diff
@@ -278,6 +286,8 @@ port: Port on which the server is running
 path: The URL path on which is checked if the server is running
 
 If the tutorial should be tested on the console environment, you have to specify a port.
+
+Note: To execute this function, devonfw-ide must be installed.
```

Comment:
We should mention that the command uses the Spring Boot Maven Plugin to run the Java application. So at the moment only Spring applications are supported